app/helpers.rb

```diff
@@ -13,6 +13,7 @@ module Helpers
       SiteTitle: 'site_title',
       TrackImage: 'track_image',
       UserProgressBar: 'user_progress_bar',
+      ExPluralizer: 'ex_pluralizer',
```

Comment:
In the future, please use Active Support's pluralizer. I will edit the PR to use active support so we can deploy this.